CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
 ## Unreleased
+## 0.16.1
+
+CHANGES:
+
+* Updated dependencies:
+  * `golang.org/x/crypto` v0.6.0 -> v0.12.0
+  * `golang.org/x/net` v0.8.0 -> v0.14.0
+  * `golang.org/x/text` v0.8.0 -> v0.12.0
+  * 
```

Comment:
Looks like an extra bullet snuck in here.

```suggestion

```

Reply:
Thanks!